### PR TITLE
Release cleanup

### PR DESCRIPTION
**Pull request by zwelz3**
Addressing comments left by sanbales on https://github.com/jupyrdf/ipyradiant/pull/82

Test procedure:
* CI passes
* Run doit locally if you want to be thorough (or check GraphExplorer example on binder). 

[Binder](https://mybinder.org/v2/gh/zwelz3/ipyradiant/release-cleanup?urlpath=lab/tree/index.ipynb)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # `ipyradiant` CHANGELOG
 
+## 0.1.2 (TODO) (2021-01-21)
```
Comment (zwelz3) [inline CHANGELOG.md]: Need to update this and include additional changes from https://github.com/jupyrdf/ipyradiant/pull/84

**Comment by nrbgt** [comment]
If you look over at #84, it pins the versions of cytoscape, pyviz_comms, etc. that work with Lab 1... the upstreams don't pin much, so this was trial-and-error.

We should likely rock that into the README, right up front...

```md
## JupyterLab compatibility

| `jupyterlab`    | `ipycytoscape`   | `pyviz_comms` |
| ----------------|------------------|---------------|
| `>=1,<2`        | `>=1.0.3,<1.1.0` | `>=1,<1.0.3`  |
```

**Comment by nrbgt** [comment]
or i can actually pin it in `setup.cfg`, which might make things smoother... but it _probably_ already works with Lab 3 (we just haven't tested it).

**Comment by zwelz3** [comment]
> If you look over at #84, it pins the versions of cytoscape, pyviz_comms, etc. that work with Lab 1... the upstreams don't pin much, so this was trial-and-error.
> 
> We should likely rock that into the README, right up front...
> 
> ```lisp
> ## JupyterLab compatibility
> 
> | `jupyterlab`    | `ipycytoscape`   | `pyviz_comms` |
> | ----------------|------------------|---------------|
> | `>=1,<2`        | `>=1.0.3,<1.1.0` | `>=1,<1.0.3`  |
> ```

45f1ebc

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # `ipyradiant` CHANGELOG
 
+## 0.1.2 (J_e) (2021-01-21)
+
+- adds `RDF2NX` transformer for converting RDF to a networkx labelled property graph.
```
Comment (nrbgt) [inline CHANGELOG.md:5]: in the future, links to issues (or PRs) better

Comment (zwelz3) [reply CHANGELOG.md:5]: I will track these better throughout the release cycle in the future.

**Review by sanbales — APPROVED** [review]
Looks great!

**Pull request merged**